src/main/java/com/minecolonies/colony/buildings/BuildingMiner.java

```diff
@@ -52,12 +56,12 @@
     /**
      * True if shaft is at bottom limit
      */
-    public                  boolean             clearedShaft            = false;
+    public boolean clearedShaft = false;
 
-    //Save in hut
+    // Save in hut
     private int startingLevelNode = 0;
-    private int active = 0;
-    private int currentLevel = 0;
+    private int active            = 0;
+    private int currentLevel      = 0;
     private BlockPos shaftStart;
 
     /**
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) First sentence should end with a period. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=checkstyle%3AJavadoc_Style)